client/src/ui/molecules/guides-menu/index.tsx

```diff
@@ -88,7 +89,7 @@ export const GuidesMenu = ({ visibleSubMenuId, toggleMenu }) => {
         Guides
       </InternalLink>
 
-      <Submenu menuEntry={menu} defaultHidden={!isOpen} />
+      <Submenu submenuId={menu.id} menuEntry={menu} defaultHidden={!isOpen} />
```

Comment:
This results in `<ul id="guides" ...>` and breaks the `#guides` TOC entry on [this page](http://localhost:3000/en-US/docs/Learn/HTML/Introduction_to_HTML#guides).

```suggestion
      <Submenu submenuId={`${menu.id}-menu`} menuEntry={menu} defaultHidden={!isOpen} />
```
(Also needs to updated in the `aria-controls` above.

Reply:
👀 I will push a fix in a second.

Reply:
✅ Resolved as follows:

1. Extracted a common `<Menu>` component for the Reference/Guides/Plus menu.
2. Added a `-menu` suffix to the id.

Reply:
^ FYI @schalkneethling